RELEASE.md

```diff
@@ -0,0 +1,137 @@
+# Releasing Habitat
+
+* Bi-weekly on Thursday
+* Notify of no-release on Wednesday
+
+## Prepare Master Branch for Release
+
+1. Pull master
+1. Create a new release branch
+1. [Update version number](#How-To:-Updating-Version-Numbers)
+1. Generate a new `CHANGELOG.md`
+
+    ```shell
+    $ export GITHUB_TOKEN=<your-token>
+    $ make changelog
+    ```
+
+1. Issue a new PR and get approval (in the form of a [dank gif](http://imgur.com/X0sNq)) from two maintainers
+1. Create & push Git tag
+
+    ```
+    $ make tag-release
+    ```
+
+## Build
+
+1. Checkout tagged release
+1. Checkout core-plans?
+1. Delete `results` directory
+1. [Build Linux Components](#how-to-build-linux-components)
+1. [Build Mac Components](#how-to-build-mac-components)
+
+## Publish
+
+1. Create release in GitHub
+1. Publish each Linux component to depot (`hab pkg upload results/*-x86_64-linux.hart`)
+1. [Release to Bintray](support/bintray-publish/README.md)
+1. Drink beer
+
+# Notification of Release (Post Release)
+
+1. Notify (Jason McDonald/Jamie Bright) to prepare Twitter/Blog messaging
```

Comment:
I didn't realize this was the context in which you were asking me the question about who is in marketing :-) Basically the Habitat Community Manager (until we get one, I'll wear that hat provisionally) will be responsible for writing and posting notifications and can coordinate with marketing if we have features we want to get extra play.